docs/source/search-runs.rst

```diff
@@ -35,13 +35,20 @@ Example Expressions
 
     attributes.status = "FINISHED"
 
-
 - Search for all failed runs.
 
   .. code-block:: sql
 
     attributes.status = "FAILED"
 
+- Search for runs created after UNIX timestamp ``1640502832``.
```

Comment:
Nit: I think we use millisecond resolution for UNIX timestamps. This appears to be in second resolution. Can we update this?

Reply:
Yikes 😬